### PR TITLE
Separate timeouts in Swarm<T> to SwarmOptions

### PR DESCRIPTION
**Pull request by earlbread**
Separate timeouts in `Swarm<T>` to `SwarmOptions` to be configurable.

### CODE REVIEW

File: Libplanet/Net/SwarmOptions.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
+using System;
+using Libplanet.Blocks;
+using Libplanet.Tx;
+
+namespace Libplanet.Net
+{
+    public class SwarmOptions
```
Comment (longfin) [inline Libplanet/Net/SwarmOptions.cs:7]: How about using `struct` instead of `class`?

Comment (earlbread) [reply Libplanet/Net/SwarmOptions.cs:7]: I made it to a `class` to use initializers. Would it be better to change it to `struct`?

File: Libplanet/Net/Swarm.cs [hunk 2]
```diff
@@ -81,6 +78,7 @@ static Swarm()
         /// <param name="trustedAppProtocolVersionSigners"><see cref="PublicKey"/>s of parties
         /// to trust <see cref="AppProtocolVersion"/>s they signed.  To trust any party, pass
         /// <c>null</c>, which is default.</param>
+        /// <param name="swarmOptions">Options for <see cref="Swarm{T}"/>.</param>
```
Comment (dahlia) [inline Libplanet/Net/Swarm.cs:81]: As it's of course swarm's options, how about simply renaming it to `options`?

Comment (earlbread) [reply Libplanet/Net/Swarm.cs]: Changed it.

File: Libplanet/Net/SwarmOptions.cs (hunk 1 above)
Comment (longfin) [reply Libplanet/Net/SwarmOptions.cs:7]: Maybe... because it seems just data class. but on the other way, it isn't a big deal.

**Pull request merged**